include/HubFramework/HUBViewModelBuilder.h

```diff
@@ -68,11 +69,19 @@ NS_ASSUME_NONNULL_BEGIN
 /**
  *  The title that the view should have in the navigation bar
  *
- *  The value of this property will be assigned to the view controller's title property
+ *  This property is an alias for `navigationItem.title`.
  */
 @property (nonatomic, copy, nullable) NSString *navigationBarTitle;
```

Comment:
Yeah, we have convenience APIs like that on `HUBComponentModel` as well, aliases to perform common tasks, such as setting `imageURLs` etc. So no need to deprecate it at this time :)